_field-types/supported-field-types/derived.md

```diff
@@ -0,0 +1,903 @@
+    },
+    "max_score": 2.8754687,
+    "hits": [
+      {
+        "_index": "logs",
+        "_id": "1",
+        "_score": 2.8754687,
+        "_source": {
+          "request": "894030400 GET /english/images/france98_venues.gif HTTP/1.0 200 778",
+          "clientip": "61.177.2.0"
+        },
+        "fields": {
+          "request": [
+            "894030400 GET /english/images/france98_venues.gif HTTP/1.0 200 778"
+          ],
+          "clientip": [
+            "61.177.2.0"
+          ],
+          "url": [
+            "/english/images/france98_venues.gif"
+          ],
+          "status": [
+            "200"
+          ]
+        }
+      },
+      {
+        "_index": "logs",
+        "_id": "3",
+        "_score": 2.8754687,
+        "_source": {
+          "request": "894250400 POST /english/venues/images/venue_header.gif HTTP/1.0 200 711",
+          "clientip": "61.177.2.0"
+        },
+        "fields": {
+          "request": [
+            "894250400 POST /english/venues/images/venue_header.gif HTTP/1.0 200 711"
+          ],
+          "clientip": [
+            "61.177.2.0"
+          ],
+          "url": [
+            "/english/venues/images/venue_header.gif"
+          ],
+          "status": [
+            "200"
+          ]
+        }
+      }
+    ]
+  }
+}
+```
+</details>
+
+Derived fields use the default analyzer set in the index analysis settings during search. You can override the default analyzer or specify a search analyzer within a search request in the same way as you do for regular fields. For more information, see [Analyzers]({{site.url}}{{site.baseurl}}/analyzers/).
+{: .note}
+
+When both an index mapping and a search definition are present for a field, the search definition takes precedence.
+{: .note}
+
+### Retrieving fields
+
+You can retrieve derived fields using the `fields` parameter in the search request in the same way as regular fields, as shown in the preceding examples. You can also use wildcards to retrieve all derived fields that match a given pattern.
+
+### Highlighting
+
+Derived fields of type `text` support highlighting using the [unified highlighter]({{site.url}}{{site.baseurl}}/opensearch/search/highlight#the-unified-highlighter). For example, the following request specifies to highlight the derived `url` field:
+
+```json
+POST /logs/_search
+{
+  "derived": {
+    "url": {
+      "type": "text",
+      "script": {
+        "source": """
+        emit(doc["request"].value.splitOnToken(" " )[2])
+        """
+      }
+    }
+  },
+  "query": {
+    "bool": {
+      "must": [
+        {
+          "term": {
+            "clientip": "61.177.2.0"
+          }
+        },
+        {
+          "match": {
+            "url": "images"
+          }
+        }
+      ]
+    }
+  },
+  "fields": ["request", "clientip", "url"],
+  "highlight": {
+    "fields": {
+      "url": {}
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+The response contains highlighting in the `url` field:
```

Comment:
"In the response, the `url` field is highlighted"? Also, should there be highlighting visible in the response?